src/stan/services/pathfinder/multi.hpp

```diff
@@ -77,6 +77,14 @@ namespace pathfinder {
  * @param[in,out] parameter_writer output for parameter values
  * @param[in,out] diagnostic_writer output for diagnostics values,
  * `error_codes::SOFTWARE` for failures
+ * @param[in] return_lp Whether single pathfinder should return lp calculations.
```

Comment:
I don't know about the name `return_lp`, since it (and this sentence, IMO) suggests the calculations are being done, and this flag just determines whether we share them, rather than the reality that it controls whether or not the log probability for each sample is even calculated.

Reply:
Yeah I think I'm going to change this to `calculate_lp`